lib/t_web/channels/feed_channel.ex

```diff
@@ -222,15 +224,13 @@ defmodule TWeb.FeedChannel do
     {:noreply, socket}
   end
 
-  # TODO refactor
-  defp render_feed_item(feed_item, screen_width) do
-    {%FeedProfile{} = profile, distance} = feed_item
-    assigns = [profile: profile, screen_width: screen_width, distance: distance]
-    render(FeedView, "feed_item.json", assigns)
+  defp render_feed_profile(profile, screen_width) do
+    assigns = [profile: profile, screen_width: screen_width]
+    render(FeedView, "feed_profile.json", assigns)
   end
 
   defp render_feed(feed, screen_width) do
-    Enum.map(feed, fn feed_item -> render_feed_item(feed_item, screen_width) end)
+    Enum.map(feed, fn feed_profile -> render_feed_profile(feed_profile, screen_width) end)
```

Comment:
I'd keep it as feed item, since feed profile doesn't really have a distance (that user didn't submit it).

Reply:
actually `feed_profile` is really a feed_profile since I deleted distance from `feed_item`

Reply:
ah, so there is only filtering now and the distance itself is not sent to the user?

Reply:
yes